### DISCUSSION TITLE
[質問]: TableViewにセルが表示されない

### DISCUSSION THREAD

**Discussion by sheep1857** [lit-iphone-course/lit-iphone-course#149] [Question-original]
### エラーの概要

Realmに保存した情報をTableViewのセルに反映したいが表示されない。

### 自分で調べた資料

これを元に作成しました。
https://qiita.com/genki_ios_dev/items/40c2651d4dc48ad52e81

### GitHubのURL

https://github.com/sheep1857/GoshuinApp.git

### TODOリスト

- [X] GitHubに最新のコードをアップロードしました。
- [X] できるだけ具体的なタイトルを設定しました。

**Comment by appleEmily** [comment]
ListTableViewController (クラス名としてはViewControllerになってる)がちゃんとstoryboardのクラスに指定されていないのが原因でした〜

築ける方法としては、ListTableViewController のViewWillAppearとかで、printしているものがよばれていないので、あれ？となってストーリーボードを確認しにいく、みたいな感じかなと思います〜！

解決方法
ListTableViewController 用のVCをstoryboardで作り直す、が一番良いかなと思います。
→今おいてくれているやつ、UITableViewControllerになっていそうなので、それだと今コードを書いているファイルを指定できないので〜。VCを作って、UITableViewを置く感じでやり直してあげると良いよ〜

breakpoint とか使うとより気付きやすいよ
breakpoint：https://qiita.com/MaShunzhe/items/b98d504b9ee6ddbd26b6


その他：
マップから探す、リストから探すをやりと、戻るボタンがないので、戻るボタンつけてあげて〜
NavigationControllerをHomeVCの前につけてあげるのも手かも。
